### ISSUE TITLE
Add an offset to the source world

### ISSUE THREAD

**Issue by el0hh** [mircokroon/minecraft-world-merger#2]
Hi, when worlds get created, they always start at 0,0 (r.0.0.mca). It's possible to add an offset x,y to one of the worlds, so 2 worlds with same coordinates can be merged together, side by side?
Regards, elo

**Comment by HoratioGamer** [comment]
I agree, this is the most important question here.   To be specific, I have one world, lets call it the root or album world that I would like to add other worlds.   The other worlds are very small patches captured in a world download of other worlds.   

I would like the program to internally change the coordinates of these small patches to be centered at 0,0, and then insert them into the root world with an offset of x,y that is specified by the user on the command line.  

java -jar world-merger.jar /path/to/world/a/ /path/to/world/b/ -album x,y

The purpose would be to allow multiplayer players to take world downloads of their builds on multi-player servers and then integrate those into like a memory album world in single player, like a photo album of everything they have built.   Then one just has to specify the album x,y of a new addition, and the coordinates are all transformed automatically.   The next time the album world is opened, and one goes to the target x,y coordinates, they would find themselves in the middle of the latest download.   

The root/album world could be superflat or anything at all, it is just the canvas on with the collection is made.    

One could call it a diary if one made repeated world downloads of builds and then placed them at advancing coordinates in the diary world -- one could document the evolution of a build.
